publications/migrations/0004_sentemaillog.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.0.5 on 2025-02-10 19:44
+
+from django.db import migrations, models
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('publications', '0003_alter_publication_timeperiod_enddate_and_more'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='SentEmailLog',
+            fields=[
```

Comment:
Is it possible to add a field that stores who triggered the email (manually from admin backend, scheduled using Django Q, ... ?